wire-ios/Wire-iOS/Sources/Managers/DatadogWrapper.swift

```diff
@@ -78,6 +78,12 @@ public class DatadogWrapper {
                 .trackBackgroundEvents()
                 .trackRUMLongTasks()
                 .enableCrashReporting(using: DDCrashReportingPlugin())
+                // Redacting an event information on tap action 
+                .setRUMActionEventMapper { actionEvent in
+                    var actionEvent = actionEvent
+                    actionEvent.action.target?.name = "ConversationListCell"
```

Comment:
don't you redact all actions here, even if there are not from ConversationListCell ?

Reply:
deleted